ui.apps/src/main/content/jcr_root/apps/acs-commons/rte-plugins/dialog-plugin/.content.xml

```diff
@@ -4,7 +4,7 @@
           jsProcessor="[default:none,min:gcc]"
           allowProxy="{Boolean}true"
           categories="acs-commons.authoring.rte-plugins.dialog-plugin.coralui2"
-          dependencies="[underscore]"/>
+          dependencies="[lodash.underscore]"/>
```

Comment:
Have you tried on AEM 6.4? Is that category available in that version?

Reply:
I haven't tested this branch on 6.4, but I do see the category on 6.4 at `/libs/clientlibs/granite/lodash/underscore`

Reply:
Same in 6.3